### PR TITLE
Cleanup docker compose

### PR DESCRIPTION
**Pull request by hardbyte**

### CODE REVIEW

File: tools/docker-compose.yml [hunk 1]
```diff
@@ -54,18 +58,6 @@ services:
     depends_on:
       - db
 
-  # The dm admin
-  db_admin:
```
Comment (wilko77) [inline tools/docker-compose.yml]: do we not need the db_admin any more?

File: base/requirements.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@ colorama==0.4.3 # required for structlog
 connexion==2.6.0
 Flask-Opentracing==1.1.0
 Flask==1.1.1
+flower==0.9.3
```
Comment (wilko77) [inline base/requirements.txt]: what is that for?

Comment (hardbyte) [reply base/requirements.txt]: they look pretty :rose:

Comment (wilko77) [reply base/requirements.txt]: true

Comment (hardbyte) [reply base/requirements.txt]: `flower` is a celery monitoring tool - we have an example use in the docker-compose file (commented out).

File: tools/docker-compose.yml (hunk 1 above)
Comment (hardbyte) [reply tools/docker-compose.yml]: I suspect no one other than myself ever used it. It is useful to see the time queries take and see the locks on the database. Want me to put it in a `developer-tools.yml` file or something?

**Pull request merged**